db/migrations/state/0013.sql

```diff
@@ -14,6 +14,9 @@ CREATE INDEX IF NOT EXISTS idx_transaction_l2_hash ON state.transaction (l2_hash
 ALTER TABLE state.batch
     ADD COLUMN wip BOOLEAN NOT NULL;
 
+ALTER TABLE state.virtual_batch
+    ADD COLUMN batch_timestamp TIMESTAMP WITH TIME ZONE NULL;
```

Comment:
```suggestion
ALTER TABLE state.virtual_batch
    ADD COLUMN final_timestamp TIMESTAMP WITH TIME ZONE NULL;
```
The name could be l1_timestamp as well
what about renaming the current timestamp column to `trusted_timestamp` or `timestamp_limit`?
Doing it, we'd have `trusted_timestamp or timestamp_limit` and `final_timestamp or l1_timestamp`

Reply:
The previous batches the batch.timestamp is the final tstamp, so I think that is confusing also

Reply:
We agree to rename it to `timestamp_batch_etrog`